### PR TITLE
Remove deprecated/disabled cloud upload.

### PR DESCRIPTION
**Pull request by apollo13**
This is a follow up to https://github.com/Yubico/yubikey-manager/commit/2bcf81b6dffc9ae6748eb251ff54a864b306e794 and restores GUI functionality again.

Untested, since I am currently not able to compile this on my own (having conflicts somewhere in the QtQuick versions)

### CODE REVIEW

File: ykman-gui/qml/OtpYubiOtpView.qml [hunk 1]
```diff
@@ -191,31 +147,17 @@ ColumnLayout {
...
+            FinishButton {
+                Layout.fillWidth: false
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
 
+                onClicked:  finish()
+                enabled: publicIdInput.acceptableInput
+                            && privateIdInput.acceptableInput
+                            && secretKeyInput.acceptableInput
+                toolTipText: qsTr("Finish and write the configuration to the YubiKey")
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
```
Comment (Jakuje) [inline ykman-gui/qml/OtpYubiOtpView.qml:160]: The qt now complains that the property is set multiple times:
```
qrc:/qml/OtpYubiOtpView.qml:160:17: Property value set multiple times
```
so I think this one can be removed.

This lead to issues described in the last comments of #357 in Fedora, where I pulled this patch to.

**Comment by apollo13** [comment]
Hi @Jakuje, since upstream no longer actively develops the project I will not try to get this working. I have switched to the new authenticator app which works fine on Fedora (if someone wants to invest time in yubikey support in Fedora it might be more beneficial to put it into packaging the new app instead of keeping the old one on life support).

**Comment by Jakuje** [comment]
No problem. I just wanted to put a comment here if somebody would like to reuse your patch for their use case to avoid spending more time debugging the same issue.

**Comment by Jakuje** [comment]
Anyway, I will add packaging of the Yubico Authenticator to my to-do list, if nobody will pick it up earlier.

**Comment by apollo13** [comment]
@Jakuje That would be lovely. I have read in the past that since it is based on flutter that there might be some issues with packaging (distant memory though, might be wrong). If there is anything I can help with, even if you just need a second set of eyes/ears feel free to ping me.

**Comment by gbcox** [comment]
> Anyway, I will add packaging of the Yubico Authenticator to my to-do list, if nobody will pick it up earlier.

I discussed the authenticator situation in Fedora here: [yubioath-desktop will not be available in the F38 Fedora repository](https://discussion.fedoraproject.org/t/f38-yubioath-desktop-will-no-longer-be-available-in-fedora-repository/45921)

**TLDR:**
    
    Yubico Authenticator is flutter based and flutter is not yet available in Fedora
    It appears that the way the package was created and bundled that it would violate current packaging guidelines

**Several Different Mitigation paths are available**

* **Supported in Fedora:** Install today [ykocli ](https://koji.fedoraproject.org/koji/packageinfo?packageID=36327). This package is a [TUI ](https://en.wikipedia.org/wiki/Text-based_user_interface) application which will mimic the functionality that was provided by *yubioath-desktop*. Please report any issues, bugs, etc. via bugzilla so they may be quickly addressed.
* **Supported Upstream:** Install the new flutter based [yubico-authenticator ](https://www.yubico.com/products/yubico-authenticator/) appimage directly from the yubico website.

For more information on **ykocli** refer to the [wiki](https://bitbucket.org/gbcox/ykocli/wiki)
